### PR TITLE
Update Person.json

### PR DESCRIPTION
**Pull request by mira-gruenwald**
added "narrative_age"

### CODE REVIEW

File: readingdata/Person.json [hunk 1]
```diff
@@ -88,6 +88,12 @@
         "required": false,
         "pattern": "(^$|NULL|S[0-9]{2}|unknown)"
       }
+    }, {
+      "name": "narrative_age",
+      "constraints": {
+        "required": false,
+        "pattern": "(^$|NULL|[0-3]|unknown)"
```
Comment (duncdrum) [inline readingdata/Person.json]:
```suggestion
        "pattern": "(^$|NULL|[0-3]|unknown)"
```
```suggestion
        "pattern": "(^$|NULL|0[0-3])"
```
I would suggest making `00` = `unkown` so to not mix text with numbers as possible field values, also use two digits (eg. `01`) so we have more room to maneuver down the road

**Review by duncdrum — CHANGES_REQUESTED** [review]
I think it ll make live easier for you if we collect all the changes for literary personas in one branch here in the csv schema repo, ie keep creating new branches off the one with the act changes, and then create pull requests against that one instead of master, once all literary changes are in we can merge them in one operation, and have just one branch name in `readingdata` to deal with

**Comment by mira-gruenwald** [comment]
ok, good thanks. I merged this branch into the one with the act changes #2 , so we have only one new branch.

**Comment by duncdrum** [comment]
yup looking good, i ll close this since the changes are in the other branch

**Pull request closed without merging**